src/Maths/Silk.NET.Maths/Simd/Simd128.Multiply.cs

```diff
@@ -0,0 +1,190 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if INTRINSICS
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+#if SSE
+using System.Runtime.Intrinsics.X86;
+#endif
+#if AdvSIMD
+using System.Runtime.Intrinsics.Arm;
+#endif
+
+namespace Silk.NET.Maths
+{
+    public static unsafe partial class Simd128
+    {
+        /// <summary>
+        /// Performs hardware-accelerated Multiply on 128-bit vectors.
+        /// </summary>
+        [MethodImpl(Scalar.MaxOpt)]
+        public static Vector128<T> Multiply<T>(Vector128<T> left, Vector128<T> right) where T : unmanaged
+        {
+
+            return Byte(left, right);            
+            [MethodImpl(Scalar.MaxOpt)]
+            static Vector128<T> Byte(Vector128<T> left, Vector128<T> right)
+            {
+                if (typeof(T) == typeof(byte))
+                {
+#if AdvSIMD
+                    if (AdvSimd.IsSupported)
+                    {
+                        return AdvSimd.Multiply(left.AsByte(), right.AsByte()).As<byte, T>();
+                    }
+#endif
```

Comment:
Please add an SSE path (see https://stackoverflow.com/questions/8193601/sse-multiplication-16-x-uint8-t)

Reply:
May I leave a todo here too? To keep PRs not too heavy

Reply:
Sure